pagopa/io-app
# feat(IDPay): Replace local IDPay feature flag with switch control

## Short description
This PR replaces the local IDPay feature flag controlled by the `.env` with a switch control in the profile section of the app. 

### Truth table

| Remote FF | Local switch | Is IDPay enabled? |
|-----------|--------------|-------------------|
| ❌         | ❌            | ❌                 |
| ❌         | ✅            | ❌                 |
| ✅         | ❌            | ❌                 |
| ✅         | ✅            | ✅                 |


https://user-images.githubusercontent.com/6160324/220319908-37a7583e-589c-44ab-b86a-91cbd79dc0da.mp4



## List of changes proposed in this pull request
- Added reducer, actions and selector to manage the flag in the store
- Added a switch control in the profile section

## How to test
- Check that no IDPay data is displayed in the wallet
- Go to the profile section
- Scroll to the bottom and enable IDPay by toggling the "Test IDPay" switch
- Restart the app and check that IDPay data is displayed correctly
